packages/browser-pool/test/new-errors.test.ts

```diff
@@ -0,0 +1,38 @@
+import { BrowserLaunchError, BrowserPool, PuppeteerPlugin } from '@crawlee/browser-pool';
+import puppeteer from 'puppeteer';
+
+describe('New errors in BrowserPool', () => {
+    const pool = new BrowserPool({
+        browserPlugins: [new PuppeteerPlugin(puppeteer, { launchOptions: { executablePath: '/dev/null' } })],
+    });
+
+    afterEach(() => {
+        delete process.env.APIFY_IS_AT_HOME;
+    });
+
+    test('they should log more information', async () => {
+        const error = await pool.newPage().catch((err) => err);
+
+        expect(error).toBeInstanceOf(BrowserLaunchError);
+
+        // Must include the executable path
+        expect(error.message).toMatch(/\/dev\/null/);
```

Comment:
nit: i guess you know it, but for matching literals you can just use strings, so you dont have to mess with escaping things for regexps

Reply:
🤦 and I even spotted that in the function signature, but I was too used to use a regex... Good point XDD